commands/signin.js

```diff
@@ -58,6 +58,7 @@ function stripProtocol (certifiedModulesUrl) {
 }
 
 function accessTokenReceived (error, response, info) {
+  if (error) { console.error(error) }
```

Comment:
this doesn't seem right.  Just prints the error (with no context on what it's from), and continues with further processing.